### PR TITLE
mapped 'false' and '0' to false for @isBoolean

### PR DESCRIPTION
**Pull request by benny-hal**

### CODE REVIEW

File: src/recipes/is-boolean.recipe.ts [hunk 1]
```diff
@@ -22,6 +23,16 @@ export function IsBooleanRecipe<Options>(
         // convert strings to boolean
         TypePropertyDecorator(() => Boolean),
 
+        // convert 'false' and '0' to false
+        TransformPropertyDecorator(({ obj, key, value }: TransformFnParams) => {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+            if (obj[key] === 'false' || obj[key] === '0') {
```
Comment (jessemyers) [inline src/recipes/is-boolean.recipe.ts]: I would use a case-insensitive comparison here

Comment (jessemyers) [reply src/recipes/is-boolean.recipe.ts]: Also, any reason you have to compare `obj[key]` instead of `value` ?

Comment (benny-hal) [reply src/recipes/is-boolean.recipe.ts]: value is already set to true when obj[key] has the string value "false"

Comment (benny-hal) [reply src/recipes/is-boolean.recipe.ts]: i did the case insensitive change

**Review by jessemyers — APPROVED** [review]

**Review by jessemyers — APPROVED** [review]

**Pull request merged**